tests/runtest.cmd

```diff
@@ -109,7 +110,12 @@ md %Core_Root%
 xcopy /s %__BinDir% %Core_Root%
 call :runtests 
 
-IF %BUILDERRORLEVEL% NEQ 0 echo Test Run  failed. Refer %__TestRunBuildLog% for details. && exit /b %BUILDERRORLEVEL%
+IF %BUILDERRORLEVEL% NEQ 0 ( 
+    echo Test Run failed. Refer to the following"
+    echo Msbuild log: %__TestRunBuildLog%
```

Comment:
this log actually shows the console output of xunit runner and steps taken to prep the run. Not building the tests